animation_nodes/nodes/mesh/get_linked_vertices.py

```diff
@@ -0,0 +1,23 @@
+import bpy
+from ... base_types import AnimationNode
+from ... data_structures import IntegerList
+
+class GetLinkedVerticesNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_GetLinkedVerticesNode"
+    bl_label = "Get Linked Vertices"
+    errorHandlingType = "EXCEPTION"
+
+    def create(self):
+        self.newInput("Mesh", "Mesh", "mesh")
+        self.newInput("Integer", "Vextex Index", "vertexIndex")
+
+        self.newOutput("Integer List", "Vertices", "vertexIndices")
+        self.newOutput("Integer List", "Edges", "edgeIndices")
+
+    def execute(self, mesh, vertexIndex):
+        if mesh is None:
+            return IntegerList(), IntegerList()
```

Comment:
Those should be `LongList`.

Reply:
Okay. Actually, the name of this node is changed during the discussion, that is the reason I update the CHANGELOG.md file, in the end, I mean when everything is finalized.

Reply:
Changed. Updated the CHANGELOG.md file.